indico/modules/categories/templates/display/overview/event.html

```diff
@@ -19,6 +19,12 @@
             {% endif %}
         <td class="title">
             <a class="js-mathjax" href="{{ event.url }}">{{ event.title }}</a>
+            {% if event.label %}
+                <span class="ui label basic mini {{ event.label.color }}"
+                      title="{{ event.label_message }}">
+                    {{- event.label.title -}}
+                </span>
+            {% endif %}
```

Comment:
I'd show it after the details - IMHO it looks a bit strange when there's e.g. the lecture speaker displayed:

![image](https://user-images.githubusercontent.com/179599/140905675-088b6fce-07e1-4897-8346-6dbde8cb4b39.png)

vs

![image](https://user-images.githubusercontent.com/179599/140905969-b290502d-56bc-4677-a81b-9f16f839618e.png)